lib/ocpp/v201/charge_point.cpp

```diff
@@ -164,7 +164,7 @@ void ChargePoint::stop() {
     this->websocket_timer.stop();
     this->client_certificate_expiration_check_timer.stop();
     this->v2g_certificate_expiration_check_timer.stop();
-    this->disconnect_websocket(websocketpp::close::status::going_away);
```

Comment:
Yes, this status was wrong (since the beginning), according to[ `websocketpp`'s documentation](https://github.com/zaphoyd/websocketpp/blob/1b11fd301531e6df35a6107c1e8665b1e77a2d8e/websocketpp/close.hpp#L78-L80):

```cpp
 /// The endpoint was "going away", such as a server going down or a browser
/// navigating away from a page.
static value const going_away = 1001;
```